docs/patterns/tools/use-tools/action-guide-without-agents.mdx

```diff
@@ -72,4 +72,39 @@ You can also execute Tools or Actions by passing in natural language prompts wit
 ```
 </CodeGroup>
 
-<br/>
\ No newline at end of file
+### How can I use Tools for a specific user?
+You can you the entity object of a specific user snd execute action on behalf of that user. Learn more about Entities [here](/patterns/Auth/connected_account#entities).
```

Comment:
There's a typo in this line: "you you" should be "you use" and "snd" should be "and"